ttnn/cpp/ttnn/operations/eltwise/unary/unary_composite.hpp

```diff
@@ -115,6 +115,14 @@ struct ExecuteUnaryCompositeOpWithFloats {
         }
 };
 
+struct ExecuteUnaryCompositeClamp {
+    static Tensor invoke(
+        const Tensor &input_tensor,
+        std::optional<float> min = std::nullopt,
```

Comment:
Can you check what is the default value for min and max? if there is a default value please update the same

Reply:
Default is None in torch.